doc/lightning-sendpay.7.md

```diff
@@ -5,7 +5,8 @@ SYNOPSIS
 --------
 
 **sendpay** *route* *payment\_hash* [*label*] [*msatoshi*]
-[*bolt11*] [*payment_secret*] [*partid*] [*localofferid*] [*groupid*] [*payment_metadata*]
+[*bolt11*] [*payment_secret*] [*partid*] [*localofferid*] [*groupid*]
+[*payment_metadata*] [*description*]
```

Comment:
```suggestion
[*bolt11*] [*payment\_secret*] [*partid*] [*localofferid*] [*groupid*]
[*payment\_metadata*] [*description*]
```

this is not a new change, at this point, I'm not really sure that `\` is required!